### PR TITLE
Takes the Tram off of the game plane, to make everything on it look a lot less flat

### PR DESCRIPTION
**Pull request by GoldenAlpharex**
## About The Pull Request
That's about it. It being on the game plane sadly had the inherent issue of "no more ambient occlusion for anything that goes in it", which made it look jarring.

Sadly, because of this, I had to put regular iron floor tiles over the pipes/wires/disposals pipes on those crossings, but that's just further motivation for LT3 and I to work on some new glass plating and actual glass tiles so we can make this work properly again.

## Why It's Good For The Game
Muh ambient occlusion.

_Seriously, it just looks *SO* much better._
image

## Changelog

:cl: GoldenAlpharex
fix: Everything that goes on the tram should look a lot less flat now!
/:cl:

### CODE REVIEW

File: code/game/objects/structures/fluff.dm [hunk 1]
```diff
@@ -274,7 +274,7 @@
 	icon = 'icons/obj/tram_rails.dmi'
 	icon_state = "rail"
 	layer = TRAM_RAIL_LAYER
```
Comment (LemonInTheDark) [inline code/game/objects/structures/fluff.dm:276]: change this value, layers are divided up by plane so this needs to go lower

**Comment by LemonInTheDark** [comment]
Might look weird with cables idk

File: code/game/objects/structures/fluff.dm (hunk 1 above)
Comment (LemonInTheDark) [reply code/game/objects/structures/fluff.dm:276]: nvm the pr I did on wallening isn't here so floor layers are intermixed with with game plane layers

**Comment by calliophis-maculiceps** [comment]
before and after might be neat

**Comment by GoldenAlpharex** [comment]
> before and after might be neat

I didn't have a "before" but I'll take a picture. The change where it was moved on the game plane is pretty recent, hence why it might not look like much has changed unless you've been playing on Tram since then.

**Comment by lessthnthree** [comment]
Here's your before:
tramconsole

**Comment by LemonInTheDark** [comment]
I hate cables

**Pull request merged**